modules/core/src/main/java/org/apache/ignite/internal/processors/query/RunningQueryManager.java

```diff
@@ -72,6 +77,18 @@
     /** Query history tracker. */
     private volatile QueryHistoryTracker qryHistTracker;
 
+    /** Number of successfully executed queries. */
+    private final LongAdderMetric successQrsCnt;
+
+    /** Number of failed queries in total by any reason. */
+    private final AtomicLongMetric failedQrsCnt;
+
+    /**
+     * Number of canceled queries. Canceled queries a treated as failed and counting twice: here and in {@link
```

Comment:
Cancelled queries count metric.
Note: Cancelled query is treated as failed one as well.